spark/dl/src/main/scala/com/intel/analytics/bigdl/visualization/README.md

```diff
@@ -8,113 +8,39 @@
 >You can find the wheel repository at https://pypi.python.org/pypi/tensorboard
 
 
-Simply, with one pip command you can install it in Ubuntu. But it is very likely that you will run into problems while installing. If you meet any error, please check the instructions below.
+Simply, with one pip command you can install it in Ununtu. But it is very likely that you will run into problems while installing. If you meet any error, please check the instructions below.
 
-If you install it with Mac OS, similar in ubuntu, it is easy to install with pip. Notice that the wheels for mac do not require CPython to support wide unicode. If you meet any error, read the instructions below carefully. You may find your own solutions from these information. 
+If you install it with Mac OS, similar in ubuntu, it is easy to install with pip. Notice that the wheels for mac do not require CPython to support wide unicode. If you meet any error, read the instructions below carefully. You may find your own solutions from these information.
+
```

Comment:
1. Qiuxin told me we have no time to test before 0.1.0-release.
2. Actually, on mac, python has to be compiled with CPython.  I think by default the python on mac is also launched by CPython.  I think the sentence about CPython on mac could be deleted. I wrote it just for a notice, because on many-linux platform it requires python2.7 to be launched by CPython which support wide unicode